dcos_tunnel/cli.py

```diff
@@ -33,6 +33,7 @@
                     [--host=<host>]
                     [--container=<container>]
                     [--client=<path>]
+                    [--docker=<docker-command>]
```

Comment:
What do you suggest? `--docker-remote-command` seemed very long. Maybe `--docker-client` since we're already using `--client` to refer to the openvpn binary? Maybe we should change client to `--openvpn-client` to make it consistent.